templates/dell/index.html

```diff
@@ -1,287 +1,331 @@
-      {{
-        image(
-          url="https://assets.ubuntu.com/v1/c4b290c8-Contact+us.svg",
-          alt="",
-          width="250",
-          height="178",
-          hi_def=True,
-          loading="lazy",
-          attrs={"class": "u-hide--small"},
-        ) | safe
-      }}
+    <div class="col-6">
+      <div>
+        <div class="p-aspect-ratio-image--at-glance">
+        </div>
+        <p>Download this summary portfolio and discover how to help your customers with Canonical and Dell joint offers.</p>
+        <a href="/engage/dell-battle-card" class="p-button">Download portfolio&nbsp;<i class="p-icon--begin-downloading"></i></a>
+        </ul>
+      </div>
+    </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row">
+    <hr class="u-no-margin--bottom u-no-margin--top" />
+    <div class="col-3">
+      <h5 class="p-text--x-small-capitalised">FROM OUR BLOG</h5>
+    </div>
+    <div class="col-9">
+      <div class="p-strip is-shallow u-no-padding--bottom">
+        <h2><a href="/blog/dell-xps-13-plus-developer-edition-now-certified-with-ubuntu-22-04-lts">Dell XPS 13 Plus Developer Edition now certified with Ubuntu 22.04 LTS</a></h2>
+        <p>
+          <a class="p-chip" href="/blog/tag/dell-xps">
+            <span class="p-chip__value">Dell XPS</span>
+          </a>
+          <a class="p-chip" href="/blog/tag/ubuntu-desktop" >
+            <span class="p-chip__value">Ubuntu Desktop</span>
+          </a>
+        </p>
+      </div>
+      <div class="p-strip is-shallow">
+        <hr class="u-no-margin--top" />
+        <h2><a href="/blog/dell-emc-poweredge-and-canonical-charmed-ceph">Dell PowerEdge and Canonical Charmed Ceph, a proven solution</a></h2>
+        <p>
+          <a href="/blog/dell-emc-poweredge-and-canonical-charmed-ceph" class="p-chip">
+            <span class="p-chip__value">Ceph</span>
+          </a>
+          <a href="/blog/tag/storage" class="p-chip">
+            <span class="p-chip__value">Storage</span>
+          </a>
+        </p>
+      </div>
+      <div class="p-strip is-shallow">
+        <hr class="u-no-margin--top" />
+        <h2><a href="/blog/dell-xps-13-plus-developer-edition-now-certified-with-ubuntu-22-04-lts">Canonical and Dell provide certified, production-ready Kubernetes solution</a></h2>
+        <p>
+          <a href="https://canonical.com/blog/canonical-and-dell-emc-provide-certified-production-ready-kubernetes-solution" class="p-chip">
+            <span class="p-chip__value">Charmed Kubernetes</span>
+          </a>
+          <a href="https://canonical.com/blog/tag/kubernetes" class="p-chip">
+            <span class="p-chip__value">Kubernetes</span>
+          </a>
+        </p>
+      </div>
+    </div>
+  </div>
+</section>
+
+
+<section class="p-strip--suru-shape-light is-deep">
+  <div class="row">
+    <hr class="u-no-padding--top u-no-padding--bottom" />
+    <div class="col-3">
+      <h5>CONTACTS</h5>
```

Comment:
This should be "contact" not "contacts". The latter implies that this section is for a list of contacts